src/Tests/Storage.fs

```diff
@@ -233,4 +235,33 @@ let tests = testList "Storage Tests" [
                 } |> ignore)
             "Can't set default tier for  Block Blobs"
     }
+    test "Sets CORS correctly" {
+        let account = storageAccount {
+            add_cors_rules [
+                StorageService.Blobs, CorsRule.AllowAll
+                StorageService.Blobs, { CorsRule.AllowAll with AllowedOrigins = Specific [ Uri "https://compositional-it.com" ] }
+                StorageService.Queues, CorsRule.create([ "https://compositional-it.com" ], [ GET ], 15, [ "exposed1"; "exposed2" ], [ "ALLOWED1"; "ALLOWED2" ] )
+            ]
+        }
+
+        let rules = (account |> findCorsResource "blobServices").properties.cors.corsRules
```

Comment:
It would be a bit clearer if these values had different names instead of shadowing each other, e.g. 'blobRules', 'blobRule1' 'blobRule2' and 'queueRules'